website/content/docs/api-gateway/api-gateway.mdx

```diff
@@ -212,15 +191,45 @@ The following table describes the required parameters for the `spec` array:
-| `image.consulAPIGateway`          | The image to use for consul-api-gateway.                                                                                                                                                                                                                                                                                                                                            | String  | `"hashicorp/consul-api-gateway:RELEASE_VERSION"` |
-| `image.envoy`                     | Specifies the container image to use for Envoy.                                                                                                                                                                                                                                                                                                                                     | String  | `"envoyproxy/envoy:v1.19-latest"`                |
-| `logLevel`                        | Specifies the error reporting level for logs. You can specify the following values: `fatal`, `error`, `warning`, `info`, `debug`, `trace`.                                                                                                                                                                                                                                          | String  | `"info"`                                         |
+| `image.consulAPIGateway`          | The image to use for consul-api-gateway. View available image tags on [DockerHub](https://hub.docker.com/r/hashicorp/consul-api-gateway/tags).                                                                                                                                                                                                                                                                                                                                           | String  | `"hashicorp/consul-api-gateway:RELEASE_VERSION"` |
+| `image.envoy`                     | Specifies the container image to use for Envoy. View available image tags on [DockerHub](https://hub.docker.com/r/envoyproxy/envoy/tags).                                                                                                                                                                                                                                                                                                                                    | String  | `"envoyproxy/envoy:RELEASE_VERSION"`                |
```

Comment:
not quite sure about the `RELEASE_VERSION` changes here, this is pinned to default to `v1.19-latest` here: https://github.com/hashicorp/consul-api-gateway/blob/9a9149ba35f6eddabfdf18900e7eaa529fb82827/internal/k8s/builder/builder.go#L29 -- the `RELEASE_VERSION` above is because the controller introspects its own version build and tries to deploy gateways with the same version tag as it by default.

Reply:
alternatively, if we are going to be super-specific, we could just update the tag above (i.e. `"hashicorp/consul-api-gateway:RELEASE_VERSION"` to `"hashicorp/consul-api-gateway:0.1.0"` and just try and make sure we update it whenever we update the CRD installation bit

Reply:
Ah okay, that makes sense, will revert this. We may want to bump the default Envoy version to at least `v1.20-latest` though.

Reply:
Opened https://github.com/hashicorp/consul-api-gateway/pull/107